go/teams/member_set.go

```diff
@@ -1,10 +1,9 @@
 package teams
 
 import (
-	"golang.org/x/net/context"
-
 	"github.com/keybase/client/go/libkb"
 	"github.com/keybase/client/go/protocol/keybase1"
+	"golang.org/x/net/context"
```

Comment:
our goimports are going to fight over this...mine will move it back to where it was.